feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss

```diff
@@ -0,0 +1,472 @@
+        align-items: flex-start;
+      }
+    }
+  }
+
+  .cx-unit-level-order-history-pagination {
+    margin-inline-start: auto;
+    @include media-breakpoint-down(sm) {
+      align-items: flex-start;
+      padding-inline-start: 1.25rem;
+      margin-inline-start: 0;
+    }
+  }
+
+  .cx-unit-level-order-history-total-result {
+    font-style: italic;
+    color: var(--cx-color-secondary);
+    padding-inline-start: 1.25rem;
+    @include media-breakpoint-down(sm) {
+      padding-bottom: 1rem;
+    }
+
+    cx-total {
+      display: flex;
+      column-gap: 0.5rem;
+    }
+  }
+
+  .cx-vertical-line-separator {
+    border-color: var(--cx-color-light);
+    border-inline-start-style: solid;
+    border-inline-start-width: thin;
+  }
+
+  .cx-unit-level-order-history-no-order {
+    display: flex;
+    flex-wrap: wrap;
+    @include type('5');
+    font-weight: $font-weight-normal;
+    min-height: 415px;
+
+    @include media-breakpoint-down(sm) {
+      min-height: 474px;
+      padding-inline-start: 1.25rem;
+      padding-inline-end: 1.25rem;
+    }
+
+    .btn {
+      margin: 1.25rem 0;
+    }
+  }
+
+  .cx-unit-level-order-history-filter-form-mobile {
+    display: none;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+
+    .cx-clear-filter {
+      flex-basis: 100%;
+      justify-content: flex-end;
+      margin-top: 2rem;
+    }
+
+    .cx-selected-filters {
+      flex-basis: 100%;
+      justify-content: flex-start;
+      margin-top: 2rem;
+    }
+
+    @include media-breakpoint-down(sm) {
+      flex-basis: 100%;
+      padding: 1rem;
+      display: flex;
+    }
+  }
+
+  .cx-unit-level-order-history-filters-show-and-remove {
+    display: flex;
+  }
+
+  .cx-unit-level-order-history-filter-form-button-block {
+    flex-basis: 100%;
+    justify-content: flex-end;
+    margin-inline-end: 0.5rem;
+    margin-bottom: 2rem;
+    align-self: center;
+
+    display: flex;
+    flex-direction: row;
+
+    .clear-btn {
+      margin-inline-end: 1rem;
+      border: none;
+      background-color: inherit;
+      font-size: var(--cx-font-size, 1rem);
+    }
+
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-unit-level-order-history-filter-form-item {
+    flex-direction: column;
+    flex-basis: 25%;
+    padding: 0.5rem;
+
+    @include media-breakpoint-down(md) {
+      flex-basis: 50%;
+      padding: 0.5rem;
+    }
+
+    @include media-breakpoint-down(sm) {
+      flex-basis: 100%;
+      padding: 0.5rem;
+      display: none;
+    }
+  }
+
+  .cx-filter-list {
+    margin: 1px 11px 1px 11px;
+  }
+
+  .cx-filter {
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    background-color: var(--cx-color-background);
+    overflow-x: hidden;
+    top: 0;
+    padding-top: 33px;
+    display: none;
+    flex-direction: column;
+    font-weight: var(--cx-font-weight-semi);
+  }
+
+  .filter-list-border {
+    border-bottom: 2px solid #e0e0e0;
```

Comment:
all colors should be variables that are declared in the _variables.scss file
this should be `var(--cx-color-light);`. `filter-list-border` seems like a class that you might reuse on another element. If an element needs a border or styling we should add the styling to it by adding a class if it doesn't have one already. You should remove this and add the border. to the specific html element with a class name following `cx-` naming.